R/endpoints.R

```diff
@@ -1205,30 +1282,36 @@ pvt_meta_norostat <- function(auth) {
   create_epidata_call("meta_norostat/", list(auth = auth), only_supports_classic = TRUE)
 }
 
-#' fetch api meta data
+#' Fetch api meta data
 #'
-#' API docs: https://cmu-delphi.github.io/delphi-epidata/api/meta.html
+#' @return An instance of epidata_call
 #'
-#' @return an instance of epidata_call
+#' @references API docs: https://cmu-delphi.github.io/delphi-epidata/api/meta.html
 #'
 #' @export
 meta <- function() {
   create_epidata_call("meta/", list(), only_supports_classic = TRUE)
 }
 
-#' fetch NIDSS dengue data
+#' Fetch NIDSS dengue data
 #'
 #' Obtains counts of confirmed dengue cases in Taiwan from Taiwan National Infectious Disease Statistical System.
 #'
-#' @details Possible location inputs can be found in <https://github.com/cmu-delphi/delphi-epidata/blob/main/labels/nidss_regions.txt> and <https://github.com/cmu-delphi/delphi-epidata/blob/main/labels/nidss_locations.txt>.
+#' @details Possible location inputs can be found in
+#' <https://github.com/cmu-delphi/delphi-epidata/blob/main/labels/nidss_regions.txt> and
+#' <https://github.com/cmu-delphi/delphi-epidata/blob/main/labels/nidss_locations.txt>.
 #'
 #' @examples
+#' \donttest{
 #' call <- nidss_dengue(locations = "taipei", epiweeks = epirange(201201, 202001))
 #' fetch_classic(call)
-#' @param locations A character string that indicates region of interest or location of interest. Possible inputs include but are not limited to (1)  nationwide, (2) central, (3) eastern, (4) kaoping, (5) northern, (6) southern, and (7) taipei.
-#' @param epiweeks Takes in the form epirange(startdate,enddate), where startdate and enddate are of the form YYYYMMDD (can be passed as string or numeric).
+#' }
+#' @param locations Character vector. Specifies the locations of interest. Possible inputs include but are not limited
+#'   to (1) nationwide, (2) central, (3) eastern, (4) kaoping, (5) northern, (6) southern, and (7) taipei.
```

Comment:
question: do we want to duplicate this information in the pkg docs + website?

Reply:
Definitely not, going to remove this and make the API doc links central